### ISSUE TITLE
[Feature Request] Upload Scanner report to SonarQube

### ISSUE THREAD

**Issue by akshay4570** [forcedotcom/sfdx-scanner#1150]
**Is your feature request related to a problem? Please describe.**
So SFDX Scanner doesn't provide an UI to look out for the static code analysis report, but SonarQube does for its enterprise version with limited PMD rules and provides a way to export reports from other 3rd party tools using the option sonar.apex.pmd.reportPaths

**Describe the solution you'd like**
The solution is SonarQube doesn't seems to identify the report generated from Scanner report, can you please provide me with a process or detailed explanation about how to go forward on this.

**Describe alternatives you've considered**
I have tried various ways on Importing the solutions to SonarQube using SFDX Scanner

**Additional context**

"Workaround": Can you do something similar with existing capabilities?

"Urgency": "Highly beneficial"

**Comment by jfeingold35** [comment]
@akshay4570 , I'm not positive I understand what the ask is here.
If you want us to output a report in a format that SonarQube can consume, that's probably doable, though we'd need to do more research to understand exactly how that would work. But if it involves anything more complicated than that, that's less likely to happen in the foreseeable future. Our team's bandwidth is really limited right now, so we probably won't be able to do complicated integrations with third-party tools.

**Comment by johnbelosf** [comment]
@akshay4570 have you tried using Code Analyzer to output in sarif format and import it into Sonarqube? https://docs.sonarsource.com/sonarqube/9.9/analyzing-source-code/importing-external-issues/importing-issues-from-sarif-reports/?gads_campaign=SQ-Hroi-PMax&gads_ad_group=Global&gads_keyword=&gclid=CjwKCAjw3oqoBhAjEiwA_UaLtm2POKaiaPSyXMnJTAvB8NNPTzK_3u_FbljLhhk-wtg34mr5gWU45hoCYgQQAvD_BwE

**Comment by stephen-carter-at-sf** [comment]
Yeah, users should be able to run code analyzer with output format of sarif. For code analyzer v5 you would do:
```
sf code-analyzer run -f results.sarif
```
and then import that results file to sonarqube: https://docs.sonarsource.com/sonarqube-server/10.6/analyzing-source-code/importing-external-issues/importing-issues-from-sarif-reports/

Closing this issue for now. If you need to re-open for any reason, feel free to do so.
